util/src/main/resources/org/killbill/billing/util/security/shiro/dao/JDBCSessionSqlDao.sql.stg

```diff
@@ -45,4 +45,19 @@ delete() ::= <<
 delete from sessions
 where id = :id
 ;
->>
\ No newline at end of file
+>>
+
+findOldActiveSessions() ::= <<
+select
+  record_id
+, id
+, start_timestamp
+, last_access_time
+, timeout
+, host
+, session_data
+from sessions
+order by record_id asc
+limit 100
```

Comment:
What if there is more than 100?

Reply:
It's just an arbitrary limit for when a very Glarge deployment with billions of rows decide to upgrade to this version (without it -- OOM) 😬The check is run every hour so the idea is that the table would be cleaned up 100 rows at a time.

There is one potential issue which is if 100 users created a session before more recent users **and** these 100 users are still active (while the recent users aren't), then the recent sessions wouldn't be cleaned-up. This is quite a corner case though. The real fix would be to `order by last_access_time asc` but I want to avoid that for the scenario described above (it would trigger a full table scan -- this could be mitigated if the team truncates the table before upgrading but if they don't read carefully the release notes...).